docs/running-local.md

```diff
@@ -0,0 +1,236 @@
+Running On a Local (Big) Machine
+================================
+
+Prerequisites
+-------------
+
+You need a beefy Linux `x86_64` machine, with `docker` and `docker-compose`
+available (and ready to use by the current UNIX user).
```

Comment:
secotrec uses them during deployment of a `ketrew+coclobas+postgre_sql+tls_tunnel` onto the public facing box. If things go wrong, then `secotrec`'s `docker-compose` exposes useful information and allow you to act on that bundle (get a shell, take them down, up, profile, etc.)